site/docs/voice/bxml/startStream.mdx

```diff
@@ -0,0 +1,419 @@
+---
+id: startStream
+title: Start Stream
+slug: /voice/bxml/startStream
+description: A general overview of Bandwidth's StartStream BXML Verb
+keywords:
+  - bandwidth
+  - voice
+  - bxml
+  - start
+  - streaming
+hide_title: false
+image: ../../static/img/bandwidth-logo.png
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The StartStream verb allows a segment of a call to be sent off to another destination for additional processing.
+
+When used on a call, audio from one or both sides (tracks) of the call will be sent to the specified destination. The stream will continue until the call ends or the [`<StopStream>`][1] verb is used. A total of 4 concurrent track streams are allowed on a call. A `<StartStream>` request that uses `both` tracks will count as 2 concurrent streams.
+A call has only two tracks, named after the direction of the media from Programmable Voice perspective:
+- `inbound`: media received by Programmable Voice from the call executing the BXML;
```

Comment:
```suggestion
- `inbound`: media, which is received by Programmable Voice from the call executing the BXML
```

Reply:
Uh, is this right? I didn't want to say that "the inbound track is media", I wanted to say what media that is. The tracks are obviously media.

Reply:
Ahhh gotcha. Sorry, I misunderstood :cold_sweat: